Exercise_1/run_sample.bash

```diff
@@ -10,7 +10,7 @@
 
```

Comment:
You might want to add this:

#####Use after training##############
##SBATCH -p express
#####################################

#####Use during training###############
#SBATCH --partition=reservation
#SBATCH --reservation=bootcamp_cpu_2023
#######################################

so it is consist with the other sessions. I just added it as well to my scripts.